### PR TITLE
Update kubeflow_assets.md

### PR DESCRIPTION
**Pull request by theadactyl**
Updating current owners/access

### CODE REVIEW

**Comment by theadactyl** [comment]
@james-jwu can you add yourself where you have access?

**Comment by james-jwu** [comment]
Sure. Do you want me to send a separate PR?

**Comment by theadactyl** [comment]
Or just add commits to this PR -- whichever is easier.

File: how-to/kubeflow_assets.md [hunk 1]
```diff
@@ -28,10 +28,7 @@ https://github.com/kubeflow-bot
 
 Admins
 
-* chrisheecho@kubeflow.org
 * ewj@kubeflow.org
-* jlewi@kubeflow.org
-* sarahmaddox@kubeflow.org
 * thea@kubeflow.org
```
Comment (james-jwu) [inline how-to/kubeflow_assets.md:32]: I think Yuan is a GSuite admin. /cc @Bobgy

**Comment by james-jwu** [comment]
I'll create a separate PR.

File: how-to/kubeflow_assets.md (hunk 1 above)
Comment (Bobgy) [reply how-to/kubeflow_assets.md:32]: Yes, and I added both @james-jwu and @paveldournov .

File: how-to/kubeflow_assets.md [hunk 2]
```diff
@@ -93,9 +89,7 @@ We have the following netlify sites
 
 Access is granted through the Netlify kubeflow-team; current owners are
 
-* jeremy@lewi.us
-* ricliu@google.com
-* saramaddoxmail@gmail.com
+* gongyuan@google.com
```
Comment (Bobgy) [inline how-to/kubeflow_assets.md:92]: @joeliedtke is also an owner

File: how-to/kubeflow_assets.md [hunk 3]
```diff
@@ -136,7 +129,6 @@ Workspace Owners:
 
 * abhishek@google.com
 * aronchick@gmail.com
-* sarahmaddox@gmail.com
 * thealamkin@google.com
```
Comment (Bobgy) [inline how-to/kubeflow_assets.md:132]: I'm also an owner, but I registered through my personal email: `gongyuan94@gmail.com`

**Pull request closed without merging**